test/constraint.jl

```diff
@@ -50,36 +50,57 @@ function constraints_test(ModelType::Type{<:JuMP.AbstractModel},
     end
 
     @testset "AffExpr constraints" begin
-        m = ModelType()
-        @variable(m, x)
+        model = ModelType()
+        @variable(model, x)
 
-        cref = @constraint(m, 2x <= 10)
-        @test JuMP.name(cref) == ""
-        JuMP.set_name(cref, "c")
-        test_constraint_name(cref, "c", JuMP.AffExpr, MOI.LessThan{Float64})
+        @testset "Scalar" begin
```

Comment:
It's easier to reproduce test failures if the testsets are standalone, i.e., a new model for each testset.